src/main/java/net/minecraftforge/client/model/ItemLayerModel.java

```diff
@@ -356,21 +398,28 @@ private int getIndex(int u, int v)
         }
     }
 
-    private static boolean isTransparent(int[] pixels, int uMax, int vMax, int u, int v)
+    private static int getAlpha(int[] pixels, int uMax, int vMax, int u, int v)
     {
-        return (pixels[u + (vMax - 1 - v) * uMax] >> 24 & 0xFF) == 0;
+        return pixels[u + (vMax - 1 - v) * uMax] >> 24 & 0xFF;
     }
 
-    private static BakedQuad buildSideQuad(VertexFormat format, Optional<TRSRTransformation> transform, EnumFacing side, int tint, TextureAtlasSprite sprite, int u, int v, int size)
+    private static boolean isTransparent(int alpha)
```

Comment:
This method should be inlined, it's kinda overkill.

Reply:
I was planning on proposing a change to `isTransparent`, so I kept it easily changeable here,